packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/nodejs/integrationsteps.md

```diff
@@ -1,64 +1,97 @@
+> **Note**: The account's username is also its primary email address.
 
-If the username and password are valid, `idx.authenticate` returns a status of `Idx.Status:PENDING`. This status indicates that an additional factor needs to be verified before the user signs in. In addition to the status, the `nextStep` field that is included in the response identifies the input parameters of the next step, which in this case is for the email authenticator key:
+### 2: The user submits their username and password
 
-```JavaScript
- status, // IdxStatus.PENDING
-  nextStep: {
-    inputs, // [{ name: 'authenticator', ... }]
-    options // [{ name: 'email', ... }, ...]
-}
-```
+When the user submits their `username` and `password`, pass them as parameters to `OktaAuth.idx.authenticate()`.
 
-The user is redirected to an authenticator list page that displays the email factor as an authenticator to be verified.
+```javascript
+const authClient = getAuthClient(req);
+const transaction = await authClient.idx.authenticate({ username, password });
+```
 
-### 2: Show the email factor in the authenticator list
+### 3: Your app displays a list of authenticators
+
+`authenticate()` returns an `IdxTransaction` object with a status of `PENDING` and `nextStep.name` of `select-authenticator-authenticate`. This indicates that the user has supplied the correct password and must select a secondary authentication factor to verify their identity.
+
+```javascript
+{
+   status: "PENDING",
+   nextStep: {
+      name: "select-authenticator-authenticate",
+      inputs: [
+         {
+         name: "authenticator",
+         type: "string",
+         },
+      ],
+      options: [
+         {
+            label: "Email",
+            value: "okta_email",
+         },
+         {
+            label: "Phone",
+            value: "okta_phone",
+         }
+      ],
+   },
+}
+```
 
-The next step is to show the email factor to the user in an authenticator list page. If not already done, build a page to display the list of authenticators from the previous step. In this use case, only the email factor appears, as shown in the following sample.
+Display all of the authenticators that the user has enrolled and are ready for use. You can find the names and IDs of the available authenticators in the `options` array.
 
 <div class="half border">
 
-![Displays a Select Authenticator page that includes an email field and a Select button.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-email-screen-verify-nodejs.png)
+![A Choose Your Authenticator page with the choices of email and phone, and a Next button](/img/oie-embedded-sdk/wireframes/choose-authenticator-email-phone-form-g2r28.png)
```

Comment:
Since this is the email use case, showing phone might confuse readers. Is there a wireframe just showing email?

Reply:
The configuration steps do not restrict to email alone, so it makes sense to have more than one option available. The text talks of a list of authenticators. I could change it to a list of email and google authenticator for instance?